tests/lib/rules/no-misleading-character-class.js

```diff
@@ -374,190 +539,342 @@ ruleTester.run("no-misleading-character-class", rule, {
             code: String.raw`var r = new RegExp("/(?<=[👍])", "")`,
             parserOptions: { ecmaVersion: 2018 },
             errors: [{
+                column: 27,
+                endColumn: 29,
                 messageId: "surrogatePairWithoutUFlag",
                 suggestions: [{ messageId: "suggestUnicodeFlag", output: String.raw`var r = new RegExp("/(?<=[👍])", "u")` }]
             }]
         },
         {
             code: String.raw`var r = new RegExp("[Á]", "")`,
             errors: [{
+                column: 22,
+                endColumn: 24,
                 messageId: "combiningClass",
                 suggestions: null
             }]
         },
         {
             code: String.raw`var r = new RegExp("[Á]", "u")`,
             errors: [{
+                column: 22,
+                endColumn: 24,
                 messageId: "combiningClass",
                 suggestions: null
             }]
         },
         {
             code: String.raw`var r = new RegExp("[\\u0041\\u0301]", "")`,
             errors: [{
+                column: 20,
+                endColumn: 38,
                 messageId: "combiningClass",
                 suggestions: null
             }]
         },
         {
             code: String.raw`var r = new RegExp("[\\u0041\\u0301]", "u")`,
             errors: [{
+                column: 20,
+                endColumn: 38,
                 messageId: "combiningClass",
                 suggestions: null
             }]
         },
         {
             code: String.raw`var r = new RegExp("[\\u{41}\\u{301}]", "u")`,
             errors: [{
+                column: 20,
+                endColumn: 39,
                 messageId: "combiningClass",
                 suggestions: null
             }]
         },
         {
             code: String.raw`var r = new RegExp("[❇️]", "")`,
             errors: [{
+                column: 22,
+                endColumn: 24,
                 messageId: "combiningClass",
                 suggestions: null
             }]
         },
         {
             code: String.raw`var r = new RegExp("[❇️]", "u")`,
             errors: [{
+                column: 22,
+                endColumn: 24,
+                messageId: "combiningClass",
+                suggestions: null
+            }]
+        },
+        {
+            code: String.raw`new RegExp("[ \\ufe0f]", "")`,
+            errors: [{
+                column: 15,
+                endColumn: 22,
+                messageId: "combiningClass",
+                suggestions: null
+            }]
+        },
+        {
+            code: String.raw`new RegExp("[ \\ufe0f]", "u")`,
```

Comment:
These two test cases are the only ones that exercise the `\\\\` check. I'm surprised it hadn't come up before.